nltk/parse/corenlp.py

```diff
@@ -328,7 +328,7 @@ def tokenize(self, text, properties=None):
 
         for sentence in result['sentences']:
             for token in sentence['tokens']:
-                yield token['originalText']
+                yield token['originalText'] or token['word']
```

Comment:
It's in token but it's empty. It's a bug upstream. I think the backoff can be removed when CoreNLP fixes it on their end.